packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -212,6 +213,26 @@ export class SummaryTreeBuilder implements ISummaryTreeWithStats {
 		this.summaryStats = mergeStats(this.summaryStats, summarizeResult.stats);
 	}
 
+	/**
+	 * This function does the following:
+	 * - For handle type result, normalizes the handles and prefixes the given 'prefix' to it.
+	 * - Adds the given summarize result to the summary tree.
+	 * - Adds the stats from the given summarize result to the overall summary stats.
+	 */
+	public addWithStatsAndPrefixHandles(
```

Comment:
@daesun-park fyi you may want to take a look at this, we are making slight changes to the way summaries are generated (the summaries themselves should still be the same, we're just trying not to leak data down here). This will maybe impact how incremental summaries are generated. I know the SharedTree team has been doing some incremental summary work

Reply:
With this change, DDSes don't need to care about what the summary path to them is when generating a `SummaryHandle`. They should put a path that is relative to the DDS which actually is simpler and more appropriate.